components/common/BaseCartItemDetails.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <div class="flex justify-between">
+      <div class="flex">
+        <div>
+          <slot name="image">
+            <BasicImage
+              :src="image"
+              :alt="name"
+              class="border image is-48x48"
+              :class="{ 'opacity-50': discarded }"
+            />
+          </slot>
+        </div>
+
+        <div class="flex flex-col justify-between ml-4 limit-width">
```

Comment:
let's remove scoped css (limit-width) and use tailwind instead

Reply:
updated